merkledag/merkledag.go

```diff
@@ -93,6 +103,23 @@ func (n *dagService) Get(ctx context.Context, c *cid.Cid) (*Node, error) {
 	return res, nil
 }
 
+func (n *dagService) GetLinks(ctx context.Context, c *cid.Cid) ([]*Link, error) {
+	node, err := n.Get(ctx, c)
+	if err != nil {
+		return nil, err
+	}
+	return node.Links, nil
+}
+
+func (n *dagService) GetOfflineLinkService() LinkService {
```

Comment:
I think this can be moved to the main `DAGService` interface. There are many situations where we would want to use this to get an offline dagservice.